src/rtutils.c

```diff
@@ -1164,7 +1164,7 @@ JL_DLLEXPORT size_t jl_static_show_func_sig(JL_STREAM *s, jl_value_t *type) JL_N
     return n;
 }
 
-JL_DLLEXPORT void jl_(void *jl_value) JL_NOTSAFEPOINT
```

Comment:
Since the inner jl_static_show is still instrumented, won't this still crash? Perhaps if ASAN is enabled, we need to compile two copies of the static show code?

Reply:
True, and the same applies to other helpers (like the LLVM dump functions). Not sure it's worth the crunch.